sensor/kubernetes/certrefresh/localscanner_tls_issuer.go

```diff
@@ -29,40 +29,41 @@ func NewLocalScannerTLSIssuer(
 	sensorNamespace string,
 	sensorPodName string,
 ) common.SensorComponent {
-	msgToCentralC := make(chan *message.ExpiringMessage)
-	msgFromCentralC := make(chan *central.IssueLocalScannerCertsResponse)
-	return &localScannerTLSIssuerImpl{
+	tlsIssuer := &localScannerTLSIssuerImpl{
 		sensorNamespace:              sensorNamespace,
 		sensorPodName:                sensorPodName,
 		k8sClient:                    k8sClient,
-		msgToCentralC:                msgToCentralC,
-		msgFromCentralC:              msgFromCentralC,
 		certRefreshBackoff:           certRefreshBackoff,
 		getCertificateRefresherFn:    newCertificatesRefresher,
 		getServiceCertificatesRepoFn: localscanner.NewServiceCertificatesRepo,
-		certRequester:                certificates.NewLocalScannerCertificateRequester(msgToCentralC, msgFromCentralC),
+		msgToCentralC:                make(chan *message.ExpiringMessage),
+		stopSig:                      concurrency.NewErrorSignal(),
 	}
+
+	tlsIssuer.certRequester = certificates.NewLocalScannerCertificateRequester(tlsIssuer.msgToCentralHandler)
```

Comment:
(solution to the blocker)

Now, assuming that we have `MessagesToCentral()` available in the requesters, the issues with the concurrency could be solved with something like this:

```go
func (i *localScannerTLSIssuerImpl) AddChannelToCentral(ch <-chan *message.ExpiringMessage)  {
	i.channels = append(channels, ch)
}
```

Now in this function (i.e., `NewLocalScannerTLSIssuer`) we would do:

```go
requester := certificates.NewLocalScannerCertificateRequester()
tlsIssuer.AddChannelToCentral(requester.MessagesToCentral())
tlsIssuer.StartForwarding() // this would need to be written
```